cursos/helm/00_solucions/99_other_wp/actividade-1-charts-helm/wp-release/templates/wp-deploy.yaml

```diff
@@ -0,0 +1,39 @@
+# wp-deploy.yaml
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
This should be a statefulset @manuelver : https://kubernetes.io/blog/2016/12/statefulset-run-scale-stateful-applications-in-kubernetes/